src/deterministic.js

```diff
@@ -26,7 +26,7 @@ Module['thisProgram'] = 'thisProgram'; // for consistency between different buil
 
 function hashMemory(id) {
   var ret = 0;
-  var len = _sbrk(0);
+  var len = sbrk(0);
```

Comment:
How does this work? I was thinking that `MANGLED_SYMBOLS` controls whether we have `_sbrk` or `sbrk`, so how can this code have only one of the two forms?

Reply:
The idea is that internally the unmangled symbols are always available and ever present.

If somebody wants unmangled symbols they are generated in addition.    I'm hoping DCE should take care the most of the cost of this.

If we didn't do it this way then all of these internal call sites would somehow have to be conditional, which seem like a bad idea.

Reply:
I see. Yeah, that makes sense. If we verify that JS optimizations remove the duplicate, then sgtm.